frontend/components/LiveQuery/TargetsInput/_styles.scss

```diff
@@ -17,9 +17,9 @@
         overflow: auto;
       }
 
-      &__data-table-block > div {
-        min-height: 89px;
-      }
+      // &__data-table-block > div {
```

Comment:
Any reason for commenting this rather than fully removing?

Reply:
Whoops! Meant to fully remove--I think I was testing if this code did anything visually but forgot to remove after testing!